doc/devel/contribute.rst

```diff
@@ -183,6 +183,22 @@ please cite us following the :doc:`/project/citing` guidelines.
 If you have developed an extension to Matplotlib, please consider adding it to our
 `third party package <https://github.com/matplotlib/mpl-third-party>`_  list.
 
+
+.. _generative_ai:
+
+
+Restrictions on Generative AI Usage
+===================================
+
+We expect authentic engagement in our community.  Do not post verbatim output
+of Large Language Models or similar generative AI as comments on GitHub or our
+discourse server as your own work.  If you use generative AI tools as an aid in
+developing code or documentation changes, ensure that you fully understand the
+proposed changes and can explain why they are the correct approach.
+
+Using machine translation of prose is acceptable.
```

Comment:
```suggestion
We expect authentic engagement in our community.  Be wary of posting output
from Large Language Models or similar generative AI as comments on GitHub or our
discourse server, as such comments tend to be formulaic and low content.  If you use 
generative AI tools as an aid in developing code or documentation changes, ensure that 
you fully understand the proposed changes and can explain why they are the correct 
approach and an improvement to the current code. 
```

Reply:
Did you mean to drop the last sentence?

Reply:
I did.  It seemed redundant with the rest.  

My overall attitude to AI is that trying to "ban" it is a game of whack-a-mole as the models evolve.  On the other hand if someone is just churning out inauthentic commentary with a low signal-to-noise, that is something we want to discourage - whether it was originally from an LLM, or from their own minds.    Using a translation tool to better express your own thoughts is clearly OK, and probably even good.